requirements/base.txt

```diff
@@ -26,5 +26,5 @@ six>=1.9.0
 sqlparse==0.1.16
 Unidecode==0.04.18
 tablib==0.11.2
-git+https://github.com/idlesign/django-sitetree.git@a626559c39ff1e865cde3441c44f42864c648dfb
+django-sitetree==1.5
```

Comment:
unfortunatelly it doesnt work
So it's issue to resolve, because menu is not generated after this.